packages/core/src/components/logo/logo.scss

```diff
@@ -0,0 +1,66 @@
+@use "~dso-toolkit/src/variables/colors";
+@use "~dso-toolkit/src/variables/media-query-breakpoints";
+@use "~dso-toolkit/src/variables/typography";
+
+:host {
+  display: block;
+  font-family: typography.$font-family-base;
+}
+
+svg {
+  display: block;
+  max-width: 100%;
+}
+
+.logo_content_omgevings {
```

Comment:
we doen geen underscore-gescheiden classnamen in de toolkit. dashes all the way.

Reply:
Het probleem van dashes is dat je niet kunt dubbelklikken om een hele selector te selecteren, met underscores werkt dat wel!